reframe/frontend/cli.py

```diff
@@ -423,6 +423,13 @@ def main():
         action='store_true',
         help='Use a login shell for job scripts'
     )
+    argparser.add_argument(
+        dest='save_report_file', default=True,
+        envvar='RFM_SAVE_REPORT_FILE',
+        configvar='general/save_report_file',
+        action='store_true',
```

Comment:
You don't need the action, since this is not a command-line option.

Reply:
Is `default=...,` needed here ?

Reply:
No, it's not, because we are setting the default in the configuration schema.

Reply:
https://github.com/eth-cscs/reframe/pull/1480/commits/cf1572eb6a77418045716888dd960846cda4416d